### PR TITLE
docs: delete and reset updates for consumer groups

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

Updates the _Configuring consumer groups_ content to include reset and delete operations from CLI and UI.

### CODE REVIEW

File: consumer-configuration/README.adoc [hunk 1]
```diff
@@ -130,17 +131,17 @@ rhoas kafka consumer-group describe [flags]
 
 For a list of consumer group commands that you can use with the CLI, see the {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^].
 
-If you're using Kafka bin scripts, you can use the `kafka-consumer-groups.sh` tool to provide the same information:
+If you're using Kafka scripts, you can use the `kafka-consumer-groups.sh` tool to provide the same information:
 [source,subs="+quotes,+attributes"]
 ----
-bin/kafka-consumer-groups.sh --bootstrap-server __<bootstrap_server>__ --describe --group my-group-id
+./kafka-consumer-groups.sh --bootstrap-server __<bootstrap_server>__ --describe --group my-consumer-group
```
Comment (PaulRMellor) [inline consumer-configuration/README.adoc:137]: @craicoverflow - I wasn't able to use the kafka-consumer-groups.sh script to perform operations like `--list`. 
Getting a  _...failed due to org.apache.kafka.common.KafkaException: Failed to find brokers to send ListGroups
_ error

File: consumer-configuration/README.adoc [hunk 2]
```diff
@@ -291,18 +292,16 @@ Alternatively, select a consumer group for a specific topic.
 --
 . In the *Consumer groups* page, select the options icon (three vertical dots) for the relevant consumer group and click *Reset Offset*.
 . Select a topic.
-. Choose a new offset position from *Latest*, *Earliest*, or *Custom offset*.
+. Choose a new offset position from *Absolute*, *Latest*, or *Earliest*.
```
Comment (rkpattnaik780) [inline consumer-configuration/README.adoc:295]: The CLI has an extra reset offset option - Timestamp. Will it be worth adding here, with a note maybe.
```
rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset timestamp --value "2016-06-23T09:07:21-07:00"
```
It is supposed to be added to UI later.

File: consumer-configuration/README.adoc (hunk 1 above)
Comment (rkpattnaik780) [reply consumer-configuration/README.adoc:137]: I am getting the same error, I guess this should be raised to teh kafka instances team.

Comment (wtrocki) [reply consumer-configuration/README.adoc:137]: was this resolved?

**Comment by wtrocki** [comment]
looks ok

File: consumer-configuration/README.adoc (hunk 1 above)
Comment (PaulRMellor) [reply consumer-configuration/README.adoc:137]: @wtrocki Not as far as I'm aware. Did you raise an issue @rkpattnaik780?

Comment (PaulRMellor) [reply consumer-configuration/README.adoc:137]: Checking expectations with Mike Edgar

File: consumer-configuration/README.adoc (hunk 2 above)
Comment (PaulRMellor) [reply consumer-configuration/README.adoc:295]: Yes. We might want to take it out when the UI reflects the option, but makes sense to add a note here.

File: consumer-configuration/README.adoc (hunk 1 above)
Comment (PaulRMellor) [reply consumer-configuration/README.adoc]: Hi @rkpattnaik780 - we were missing the `--command-config` flag in the commands. The scripts seem to work now. Even created a new consumer group with the consumer script. :+1: 

I've added a new sub-section that describes the setup needed for the Kafka scripts. And updated the example commands.

**Review by rkpattnaik780 — COMMENTED** [review]
Looks good, just a minor typo.

**Review by smccarthy-ie — COMMENTED** [review]
LGTM!

**Review by rkpattnaik780 — APPROVED** [review]

**Comment by PaulRMellor** [comment]
@wtrocki Hi Wojciech. If you have no further comments, I guess we can merge.

**Pull request merged**